integration_tests/test_basic.py

```diff
@@ -334,10 +334,10 @@ def test_transaction(cronos):
     with concurrent.futures.ThreadPoolExecutor(4) as executor:
         futures = []
         futures.append(
-            executor.submit(contracts["test_revert_1"].transfer, 5 * (10**18) - 1)
+            executor.submit(contracts["test_revert_1"].transfer, 5 * (10 ** 18) - 1)
```

Comment:
You need to update the `black` version to latest, otherwise, it'll fail the one in CI.

Reply:
Do I change it from here?
https://github.com/macong-cdc/cronos/blob/test-revert-contract-call/integration_tests/pyproject.toml#L12

Or should I change it at somewhere else?

Reply:
I mean your dev env

Reply:
The repo is using 22.6 already: https://github.com/macong-cdc/cronos/blob/test-revert-contract-call/integration_tests/poetry.lock#L91